### ISSUE TITLE
`next_/prev_rateRange` control to cycle through `Preferences > Decks > Slider Range` options

### ISSUE THREAD

**Issue by Swiftb0y** [mixxxdj/mixxx#11068]
### Feature Description

Often times controllers seem to have a way of displaying the current rateRange but are flexible in the ranges they allow. They often just copy the ranges defined in the Deck preferences and implement their own cycling logic, but that comes with usual disadvantages of unnecessarily duplicated code. So I'd like to propose a `next_rateRange` and `prev_rateRange` control (similar to `next_`/`prev_chain`). 

Mappings which currently implement the same raterange cycling logic are (non exhaustive list):
* https://github.com/mixxxdj/mixxx/blob/0cf07a192e93d7c02fa62273879d1b9f5c8bcde2/res/controllers/Denon-MC7000-scripts.js#L60
* https://github.com/mixxxdj/mixxx/blob/0cf07a192e93d7c02fa62273879d1b9f5c8bcde2/res/controllers/Numark-N4-scripts.js#L27
* https://github.com/mixxxdj/mixxx/blob/0cf07a192e93d7c02fa62273879d1b9f5c8bcde2/res/controllers/Numark-V7-scripts.js#L16
* https://github.com/mixxxdj/mixxx/blob/0cf07a192e93d7c02fa62273879d1b9f5c8bcde2/res/controllers/American-Audio-VMS2-scripts.js#L32

**Comment by ronso0** [comment]
I agree. Additionally, some mappings allow a range of 100% even though 90% is the maximum in the preferences.
And at least the Reloop TerminalMix even calculates the new pitch value to keep the slider in sync, while actually the pitch shouldn't change and softTakeover take care of sync'ing.

**Comment by daschuer** [comment]
How does this idea play with the Utrapitch PR? 
#1767
Maybe we find a solution that covers both.
Something like stepping through the ultra pitch slider?

**Comment by ronso0** [comment]
IIUC the new next/prev control would be the same as picking a new value in the preferences.
Just test it with #1767 

We didn't even find a common way to map ultra-pitch, so let's not worry about that here.
